addons/account_peppol/wizard/service_wizard.py

```diff
@@ -0,0 +1,109 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+
+_logger = logging.getLogger(__name__)
+
+
+class PeppolService(models.TransientModel):
+
+    _name = 'account_peppol.service'
+    _order = 'document_name'
+    _description = 'Peppol Service'
+
+    wizard_id = fields.Many2one('account_peppol.service.wizard')
+    document_identifier = fields.Char()
+    document_name = fields.Char()
+    enabled = fields.Boolean(default=False)
```

Comment:
```suggestion
    enabled = fields.Boolean()
```
`False` is already the default boolean value